### PR TITLE
Add README.md for bin2txt converter

### PR DESCRIPTION
**Pull request by ldorau**
This change is [Reviewable](https://reviewable.io/reviews/pmem/vltrace/39)

### CODE REVIEW

**Comment by krzycz** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/vltrace/39)*

File: tools/bin2txt/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+so the following files:
+
+- converter.py
+- exceptions.py
+- syscallinfo.py
+- syscall.py
+- syscalltable.py
+- utils.py
+
+have to be identical in both projects:
```
Comment (marcinslusarz) [inline tools/bin2txt/README.md]: "should be kept in sync in these projects:" sound better IMHO

File: tools/bin2txt/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+1) https://github.com/pmem/vltrace/tree/master/tools/bin2txt
+2) https://github.com/pmem/pmemfile/tree/master/src/tools/antool
+
+Keep in mind to synchronize them!
```
Comment (marcinslusarz) [inline tools/bin2txt/README.md]: And then this is not needed.

**Comment by ldorau** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[tools/bin2txt/README.md, line 16 at r1](https://reviewable.io:443/reviews/pmem/vltrace/39#-KwFBYOtFkCLSU5WMNBa-r1-16:-KwFlFlBKGiZ0hVmH9mI:b-896fix) ([raw file](https://github.com/pmem/vltrace/blob/49ce2e0abd6581f255217462acc79089963317a9/tools/bin2txt/README.md#L16)):*
<details><summary><i>Previously, marcinslusarz (Marcin Ślusarz) wrote…</i></summary><blockquote>

"should be kept in sync in these projects:" sound better IMHO
</blockquote></details>

Done.

---

*[tools/bin2txt/README.md, line 21 at r1](https://reviewable.io:443/reviews/pmem/vltrace/39#-KwFBYOtFkCLSU5WMNBa-r1-21:-KwFlG90jy_6yQy4Uv1h:b-896fix) ([raw file](https://github.com/pmem/vltrace/blob/49ce2e0abd6581f255217462acc79089963317a9/tools/bin2txt/README.md#L21)):*
<details><summary><i>Previously, marcinslusarz (Marcin Ślusarz) wrote…</i></summary><blockquote>

And then this is not needed.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/vltrace/39)*

**Comment by marcinslusarz** [comment]
:lgtm:

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/vltrace/39#-:-KwFo5Zcczab93g0odgG:bnfp4nl)*

**Comment by marcinslusarz** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/vltrace/39)*

**Pull request merged**